ua-frontend/libs/ua-core/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@ultimate-app/core",
```

Comment:
Not that we are publishing this yet, but are you assuming that we will grab the @ultimate-app npm namespace?